scripts/developer_scripts/country_ordering/Belgium_variants.txt

```diff
@@ -0,0 +1,147 @@
+Antwerp	Antwerpen
+Ghent	Gent
+Bruges	Brugge
+Ypres	Ieper
+Ostend	Oostende
+Anvers	Antwerpen
+Baerle-Duc	Baarle-Hertog
+Lierre	Lier
+Malines	Mechelen
+Bevekom	Beauvechain
+Eigenbrakel	Braine-l'Alleud
+Kasteelbrakel	Braine-le-Château
+Genepien	Genappe
+Graven	Grez-Doiceau
+Itter	Ittre
+Geldenaken	Jodoigne
+Terhulpen	La Hulpe
+Nijvel	Nivelles
+Perwijs	Perwez
+Tubeke	Tubize
+Waver	Wavre
+Aat	Ath
+'s Gravenbrakel	Braine-Le-Comte
```

Comment:
Just browsing through this PR and I think this line might have a typo? It starts with `'s`

Reply:
Nevermind I see this in other places, I'm just uncultured apparently :smile:

Reply:
Haha I totally would have agreed with you and fixed it! Oh Belgium :)